contexts/multikey-v1.jsonld

```diff
@@ -13,6 +13,10 @@
           "@id": "https://w3id.org/security#controller",
           "@type": "@id"
         },
+        "expires": {
+          "@id": "https://w3id.org/security#expires",
```

Comment:
```suggestion
          "@id": "https://w3id.org/security#expiration",
```

Reply:
+1 @HelgeKrueger -- the expansion of `key.expires` into `https://w3id.org/security#expiration` may seem odd, but is needed for backwards compatibility.